README.md

```diff
@@ -7,8 +7,11 @@ curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | b
 2. Restart your bash session and install node using NVM: `nvm install node`
 3. In this directory, install the node packages, including webpack: `npm install`
 
+## Testing the App
+- Run `npm test`. This will lint your test files, and then run them.
+
 ## Running the App
-- Run `npm run build`.  This will start webpack, which will "watch" the files in `src/` and will update the bundle automatically.  This will also run eslint whenever your code changes so you can see if the linter fails.
+- Run `npm run build`.  This will start webpack, which will "watch" the files in `src/` and will update the bundle automatically.  This will also run eslint whenever your code changes so you can see if the linter fails. It will also automatically run unit tests whenever you save a file it is watching.
```

Comment:
nope I made a mistake and left an import behind you don't need. Thanks for testing this. will fix